src/Features/Core/Portable/Completion/Providers/AbstractDocCommentCompletionProvider.cs

```diff
@@ -270,19 +268,8 @@ private static string FormatParameterRefTag(string kind, string name)
 
         public override async Task<CompletionChange> GetChangeAsync(Document document, CompletionItem item, char? commitChar = null, CancellationToken cancellationToken = default)
         {
-            var includesCommitCharacter = true;
-
-            if (commitChar == ' ' &&
-                XmlDocCommentCompletionItem.TryGetInsertionTextOnSpace(item, out var beforeCaretText, out var afterCaretText))
-            {
-                includesCommitCharacter = false;
-            }
-            else
-            {
-                beforeCaretText = XmlDocCommentCompletionItem.GetBeforeCaretText(item);
-                afterCaretText = XmlDocCommentCompletionItem.GetAfterCaretText(item);
-            }
-
+            var beforeCaretText = XmlDocCommentCompletionItem.GetBeforeCaretText(item);
+            var afterCaretText = XmlDocCommentCompletionItem.GetAfterCaretText(item);
```

Comment:
The behavior triggered on space that was relying on this code was removed in [https://github.com/dotnet/roslyn/pull/21479](https://github.com/dotnet/roslyn/pull/21479). Just want to point it out in case someone else who's reviewing this PR also wondered about this

Reply:
Good to know. Thanks for the ptr.